.github/workflows/release.yml

```diff
@@ -0,0 +1,63 @@
+name: Build and Release
+
+on:
+  push:
+    tags:
+      - '*.*'
+    paths-ignore:
+    - 'docs/**'
+    - '.github/site.yml'
```

Comment:
what is this?

Reply:
avoids this file.  https://github.com/IBM/FHIR/blob/master/.github/workflows/site.yml and site changes on build

Reply:
don't you need workflows in the path then?

Reply:
fixed